src/tests/ftest/control/dmg_telemetry_io_latency.py

```diff
@@ -315,13 +315,15 @@ def verify_io_dtx_committed_metrics(self, baseline_data, metrics_data, test_metr
         # test ops = (create dir + punch obj + punch dir) * repetitions
         test_ops *= repetitions
         num_operations += test_ops
+        # 2 DTX entries on 2 replicas
+        num_operations *= 2
```

Comment:
Instead of hard-coding the number of replicas, could we import the number of servers from the yaml file?